### PR TITLE
Updated tests for changes in PR https://github.com/Azure/azure-rest-api-specs/pull/18609

### PR DESCRIPTION
**Pull request by D1v38om83r**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Comment by D1v38om83r** [comment]
SDK generated from https://github.com/Azure/azure-rest-api-specs/pull/18609/commits/f830443fc02f57330ea9dc27a37e9f4bc6ca8010

PR https://github.com/Azure/azure-rest-api-specs/pull/18609

File: eng/mgmt/mgmtmetadata/compute_resource-manager.txt [hunk 1]
```diff
@@ -3,13 +3,13 @@ AutoRest installed successfully.
 Commencing code generation
 Generating CSharp code
 Executing AutoRest command
```
Comment (grizzlytheodore) [inline eng/mgmt/mgmtmetadata/compute_resource-manager.txt:5]: please remove this from the PR

File: sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/GalleryApplicationVersionPublishingProfile.cs [hunk 2]
```diff
@@ -59,13 +59,17 @@ public GalleryApplicationVersionPublishingProfile()
...
+        public GalleryApplicationVersionPublishingProfile(UserArtifactSource source, IList<TargetRegion> targetRegions = default(IList<TargetRegion>), int? replicaCount = default(int?), bool? excludeFromLatest = default(bool?), System.DateTime? publishedDate = default(System.DateTime?), System.DateTime? endOfLifeDate = default(System.DateTime?), string storageAccountType = default(string), string replicationMode = default(string), IList<GalleryTargetExtendedLocation> targetExtendedLocations = default(IList<GalleryTargetExtendedLocation>), UserArtifactManage manageActions = default(UserArtifactManage), UserArtifactSettings settings = default(UserArtifactSettings), IDictionary<string, string> advancedSettings = default(IDictionary<string, string>), bool? enableHealthCheck = default(bool?))
             : base(targetRegions, replicaCount, excludeFromLatest, publishedDate, endOfLifeDate, storageAccountType, replicationMode, targetExtendedLocations)
         {
             Source = source;
             ManageActions = manageActions;
+            Settings = settings;
```
Comment (grizzlytheodore) [inline sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/GalleryApplicationVersionPublishingProfile.cs:71]: customization needed - for theo

File: sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/SdkInfo_ComputeManagementClient.cs [hunk 3]
```diff
@@ -71,5 +71,16 @@ internal static partial class SdkInfo
               }.AsEnumerable();
           }
       }
+      // BEGIN: Code Generation Metadata Section
```
Comment (grizzlytheodore) [inline sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/SdkInfo_ComputeManagementClient.cs:74]: please also remove this from the PR

**Review by grizzlytheodore — CHANGES_REQUESTED** [review]

File: eng/mgmt/mgmtmetadata/compute_resource-manager.txt (hunk 1 above)
Comment (D1v38om83r) [reply eng/mgmt/mgmtmetadata/compute_resource-manager.txt]: done

File: sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/SdkInfo_ComputeManagementClient.cs (hunk 3 above)
Comment (D1v38om83r) [reply sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/SdkInfo_ComputeManagementClient.cs]: done

**Pull request merged**